api/libproject/devstart.rb

```diff
@@ -178,9 +178,25 @@ def format_benchmark(bm)
   "%ds" % [bm.real]
 end
 
-def dev_up()
+def dev_up(cmd_name, args)
```

Comment:
We already have this: `run-api`. It should probably be renamed to be more clear for its purpose.

I think it would make sense as part of this PR to either delete run-api or make one call the other.